doc/MANUAL.md

```diff
@@ -637,6 +637,10 @@ haxx,curl,7.34.0,CVE-2014-0139,MEDIUM,Unexplored,
 
 4. `--format html` - creates a report in html format according to the specified HTML theme.
 
+The unexplored and new CVEs will be highlighted, it will look something like this:
+
+![image](https://i.imgur.com/5G1XF8Y.png)
```

Comment:
Similar to the other PR: please add the image into `doc/` so we don't have to rely on external servers.  Could be in a sub-directory of `doc/` if that looks nicer.